### DISCUSSION TITLE
Plot settings for line thickness and point size

### DISCUSSION THREAD

**Discussion by valschmidt** [foxglove/community#207] [Q&A]
In an operational setting, where visibility of a plot is critical, one needs the ability to increase the width of the lines and size of points drawn in the plot windows. To some extent this is true of the plot labels, although one can usually fix the bounds and learn to interpret the plot quickly. We need to be able to see it clearly from a row of seats behind the robot operator sitting at the computer. Critically, specification of these formatting parameters need to be saved with the panel layout.

**Comment by defunctzombie** [comment]
We have the ability to customize point and line size: https://docs.foxglove.dev/docs/visualization/panels/plot/#series
